deploy/crds/mongodb.com_v1_mongodb_custom_role.yaml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
I added a file to illustrate how one could potentially define a custom role via the CRD- not sure if we need one, or if having this explained in the docs would be sufficient. Thoughts?

Reply:
I am absolutely in favour of having examples for any new feature we implement

Reply:
the more examples the better!